### PR TITLE
Generate QRcode view

### PR DESCRIPTION
**Pull request by erkkisula**

### CODE REVIEW

File: app/src/main/java/eu/niggas_with_attitude/qrify/MainActivity.java [hunk 1]
```diff
@@ -27,12 +29,20 @@ protected void onCreate(Bundle savedInstanceState) {
...
                         .setOrientationLocked(false)
                         .setCaptureActivity(ScannerActivity.class)
                         .initiateScan()
         );
+
+        openGeneratorButton.setOnClickListener(view -> startActivity(new Intent(MainActivity.this, GeneratorActivity.class)));
+
+        // Used so that temporary images can be saved
+        StrictMode.VmPolicy.Builder builder = new StrictMode.VmPolicy.Builder();
+        StrictMode.setVmPolicy(builder.build());
```
Comment (KareVulli) [inline app/src/main/java/eu/niggas_with_attitude/qrify/MainActivity.java:45]: I would move this to GeneratorActivity, so everithing related to QR generation is in one place. Or better yet, use FileProvider API to get content:// URI, 

 Disabling VmPolicy restrictions seems an easy way out, that could backfire us later. When debugging crashes or freezes for instance.

Comment (erkkisula) [reply app/src/main/java/eu/niggas_with_attitude/qrify/MainActivity.java]: Switched to FileProvider

File: app/src/main/java/eu/niggas_with_attitude/qrify/GeneratorActivity.java [hunk 2]
```diff
@@ -2,13 +2,15 @@
...
 import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.content.FileProvider;
 
 
 import android.content.Intent;
 import android.graphics.Bitmap;
 import android.net.Uri;
 import android.os.Bundle;
 import android.os.Environment;
+import android.os.StrictMode;
```
Comment (KareVulli) [inline app/src/main/java/eu/niggas_with_attitude/qrify/GeneratorActivity.java:13]: I think this import is unnecessary.
```suggestion
```

File: app/src/main/java/eu/niggas_with_attitude/qrify/MainActivity.java (hunk 1 above)
Comment (erkkisula) [reply app/src/main/java/eu/niggas_with_attitude/qrify/MainActivity.java]: Removed import

**Pull request merged**